presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOracleTypeMapping.java

```diff
@@ -564,8 +564,8 @@ private Session number(UnsupportedTypeHandling unsupportedTypeHandlingStrategy,
     @Test
     public void testSpecialNumberFormats()
     {
-        oracleServer.execute("CREATE TABLE test (num1 number)");
-        oracleServer.execute("INSERT INTO test VALUES (12345678901234567890.12345678901234567890123456789012345678)");
+        getOracleSqlExecutor().execute("CREATE TABLE test (num1 number)");
```

Comment:
> Do not use oracleServer explicitly

This is err prone.

maybe we create explicit superclass which does not have `oracleServer` field, where the test live

Reply:
Good point. Changed.